shell/components/__tests__/SimpleBox.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { mount } from '@vue/test-utils';
+import SimpleBox from '@shell/components/SimpleBox.vue';
+
+describe('component: SimpleBox.vue', () => {
+  const wrapper = mount(SimpleBox, { propsData: { title: 'Simple box title' } });
+
+  it('show title', () => {
+    const title = wrapper.find(`[data-testid="simple-box-title"]`);
+
+    expect(title.element).toBeDefined();
+  });
+
+  it('show close button', async() => {
+    await wrapper.setProps({ canClose: true });
+    const closeButton = wrapper.find(`[data-testid="simple-box-close"]`);
+
+    expect(closeButton.element).toBeDefined();
+  });
+
+  it('close emit', async() => {
+    await wrapper.setProps({ canClose: true });
+    const closeButton = wrapper.find(`[data-testid="simple-box-close"]`);
+
+    await closeButton.trigger('click');
+
+    expect(wrapper.emitted('close')).toBeTruthy();
```

Comment:
same as previous comment

Reply:
Thanks for the review Alex, got you points, it's a good idea to be more specific in verifying expected results.